website/content/maintainers/community/contributor-growth-framework/pr-workflow.md

```diff
@@ -3,7 +3,6 @@ title: "A successful PR workflow"
 linkTitle: "PR Workflow"
 date: 2020-03-29
 weight: 2
-draft: true
 description: >
```

Comment:
`User opens an issue (bug, feature request, vulnerability)`- for vulnerabilities, there is usually a special process. Where the issue can be reported to the project maintainers privately, without exposing the details - and with the implicit way of exploiting it - to the public. Then after the issue is fixed, and released, the project owners publish the information to the public, and recommend the upgrade. Not sure what would be the right place to put this recommendation

Reply:
We have added a link to the TAG Security templates and processes for how projects should handle security reports.